TEST.md

```diff
@@ -194,6 +194,68 @@ To cleanup the namespaces after the test has finished, run:
 $ bin/test-cleanup
 ```
 
+### Testing the dashboard
+
+We use WebdriverIO to test how the web dashboard looks and operates locally in
+Chrome. For cross-browser testing, we use SauceLabs, which runs simulataneous
+tests on different browsers in the cloud.
+
+Whether you are running the tests locally or in the cloud, you'll first run
+`bin/web dev` which will start up the dashboard at `localhost:7777`.
+
+To run a local WebdriverIO instance that will run the tests on a local instance
+of Chrome, run:
+
+```bash
+$ bin/web integration local
+```
+
+To run cross-browser tests via SauceLabs, you need to do a few things first:
+
+1. Sign up for a (free) SauceLabs sub-account for the account 'buoyant'. If you
+   are not a Buoyant staffer, the best way to get an account invite is to ask in
+   the [Linkerd Slack channel](https://slack.linkerd.io).
+
+2. Once you have your username and key, set them as permanent environment
+   variables. This keeps your credentials private, and means that everyone on
+   the team can run the tests via their unique login without modifying the
+   authentication credentials in the test files. Open your `~/.bash_profile`
+   file and add:
+
+```
```

Comment:
formatting nit: prefix this block with `bash`, and also indent it 4 spaces to align it with the list item. similar comment for item 3 below.